prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -555,21 +559,32 @@ class FHIRRouter(
     /**
      * Log the results of running filters (referenced by the given [filterName]) on items out of a [report] during the
      * "route" step for a [receiver], tracking the [filterType] and tying the results to a [receiver] and [bundle].
+     * @param filterName Name of evaluated filter
+     * @param bundle FHIR Bundle that was evaluated
+     * @param report Report object passed for logging purposes
+     * @param receiver Receiver of the report
+     * @param filterType Type of filter used
+     * @param focusResource Starting point for the evaluation, can add additional data to the log
      */
     internal fun logFilterResults(
         filterName: String,
         bundle: Bundle,
         report: Report,
         receiver: Receiver,
-        filterType: ReportStreamFilterType
+        filterType: ReportStreamFilterType,
+        focusResource: Base? = null
     ) {
+        var filteredTrackingElement = bundle.identifier.value ?: ""
+        if (focusResource != null) {
+            filteredTrackingElement += " at " + focusResource.idBase
```

Comment:
@JessicaWNava the result of this is a filter log message like this: `For ignore.FULL_ELR, filter [%obsPerformedCodes.intersect('123-4').exists()][] filtered out item MT_IGCAA_ORU_AAPHELR.1.6214638 at Observation/1685109567290401000.51a5d5cd-f839-4161-af20-315a2e526a1d`. Is that the kind of ID you were referring to in the description on [the ticket](https://github.com/CDCgov/prime-reportstream/issues/8689)? I'm a little wary of using that, since it only really carries meaning in the context of our internal FHIR bundle (or if the report was FHIR to begin with). Still helps if you pull down the FHIR bundle from object storage to look up that observation, but I'm wondering if instead using the LOINC code would be more useful. Any thoughts?

Reply:
I'm fine waiting to merge until we're 100% sure of what will be the most useful log message content

Reply:
How difficult would it be to use the observation id and loinc code? Honestly, this was a ticket Carlos had me write up. I am not sure what he had in mind, so it's really kind of a "use our best judgement" sort of thing. The goal is for the sender to be able to tell which one didn't make it through

Reply:
Using the LOINC code is not too bad when we know it's an Observation. Made updates to use it. Additionally, it shows the codesystem being used. e.g. `"For ignore.FULL_ELR, filter [%obsPerformedCodes.intersect('123-4').exists()][] filtered out item MT_IGCAA_ORU_AAPHELR.1.6214638 with http://loinc.org code:19000-9",`